package.json

```diff
@@ -15,24 +15,34 @@
   "license": "MIT",
   "packageManager": "yarn@1.22.19",
   "scripts": {
+    "inst": "run-s -c inst:*",
+    "inst:lib": "yarn install",
+    "inst:ex": "CMD=inst run-s -c run-ex:*",
```

Comment:
nits: If you set it to `CMD=install`, won't it work without `inst` in `examples`?